generate.sh

```diff
@@ -68,20 +68,47 @@ for f in "$srcdir"/{,.}*.mustache; do
                 continue
             fi
             ;;
-	config.yml)
-	    mustache='{{ circleci }}'
-	    bool=$(get_yaml meta.yml <<<"$mustache")
+        config.yml)
+	        mustache='{{ circleci }}'
+	        bool=$(get_yaml meta.yml <<<"$mustache")
+                if [ -n "$bool" ] && [ "$bool" != false ]; then
+                    mkdir -p -v .circleci && target=".circleci/$target"
+                else
+                    continue
+                fi
+                ;;
+        default.nix)
+            mustache='{{ nix }}'
+            bool=$(get_yaml meta.yml <<<"$mustache")
             if [ -n "$bool" ] && [ "$bool" != false ]; then
-                mkdir -p -v .circleci && target=".circleci/$target"
+                : noop
             else
                 continue
             fi
             ;;
-        default.nix)
+        shellHook.sh)
             mustache='{{ nix }}'
             bool=$(get_yaml meta.yml <<<"$mustache")
             if [ -n "$bool" ] && [ "$bool" != false ]; then
-                : noop
+                mkdir -p -v .nix && target=".nix/$target"
+            else
+                continue
+            fi
+            ;;
+        config.nix)
+            mustache='{{ nix }}'
+            bool=$(get_yaml meta.yml <<<"$mustache")
+            if [ -n "$bool" ] && [ "$bool" != false ]; then
+                mkdir -p -v .nix && target=".nix/$target"
+            else
+                continue
+            fi
+            ;;
+        nixpkgs.nix)
```

Comment:
Then we could remove this file from the templates? (It doesn't seem like something that should be present in the templates.)